### PR TITLE
chore(docs): restructure and rewrite content in documentation and quality standards chapters

### PR DESCRIPTION
**Pull request by BirgitBader**
**Context**

- Closes [issue 15](https://github.com/otto-ec/techwriting_hub/issues/15)
- Removes accordions to better display the content hidden behind them.
- Uses TOCs to access the different sub-sections.
- Rewrites and trims documentation chapter.

**To reviewers**

- Check if the restructuring helps to better digest and access the provided content.
- Check if rewriting leads to optimization of the chapters.
- Fix any typos or grammar you might find.
- Add anything that you see missing.

### CODE REVIEW

**Comment by BirgitBader** [comment]
@thake It would be great if you could review this :)

File: api-guidelines/global/core-principles/quality-standards.md [hunk 1]
```diff
@@ -46,30 +54,27 @@ We are conservative in exposing our data and the [Principle of least privilege](
...
 
-:::: accordion Documentation & Support
-We help both vendors during development and users of our API with the integration by offering suitable ways of exchange and support.
-As the primary resource for explaining our API and its capabilities, documentation must be as accessible to the audience as possible.
-We provide all consumers of our API with comprehensive, professional, up-to-date, and complete information.
-::::
+## Documentation and support
 
-:::: accordion Communication
+We help both vendors during development and users of the API with the integration by offering suitable ways of exchange and support.
```
Comment (cgebken) [inline api-guidelines/global/core-principles/quality-standards.md:65]: What is a "vendor" and what is the difference to "provider"?

File: api-guidelines/global/core-principles/quality-standards.md [hunk 2]
```diff
@@ -46,30 +54,27 @@ We are conservative in exposing our data and the [Principle of least privilege](
...
+As the primary resource for explaining the API and its capabilities, documentation must be as accessible to the audience as possible.
+We provide all consumers of the API with comprehensive, professional, up-to-date, and complete information.
 
-We always keep both developers and consumers of our API informed through appropriate channels.
+## Communication
+
+We always keep both developers and consumers of the API informed through appropriate channels.
```
Comment (cgebken) [inline api-guidelines/global/core-principles/quality-standards.md:71]: Has there been a specific reason for "developers" here?

```suggestion
We always keep both providers and consumers of the API informed through appropriate channels.
```

File: api-guidelines/global/core-principles/quality-standards.md [hunk 3]
```diff
@@ -46,30 +54,27 @@ We are conservative in exposing our data and the [Principle of least privilege](
...
+## Developer experience
+
+API consumers should have fun using the API.
 Our goal is to provide seamless experience to developers when writing software, and to increase their efficiency.
-API consumers should be comfortable using our API in their programming language of choice, finding the functionality they need, as well as using the output.
-We give developers the right tools to help them succeed and aim to provide an as short as possible TTFHW.
-::::
-:::::
+API consumers should be comfortable using the API in their programming language of choice, finding the functionality they need, as well as using the output.
+We give developers the right tools to help them succeed and aim to provide an as short as possible TTFHW.
```
Comment (cgebken) [inline api-guidelines/global/core-principles/quality-standards.md:80]: I had to look up TTFHW... usually we do not want to use abbreviations without introducing them first... Is it such a well-known concept?

File: api-guidelines/global/core-principles/documentation.md [hunk 4]
```diff
@@ -1,57 +1,39 @@
...
-::: accordion Code examples
-**Default values**  
-Request and response examples that belong to every endpoint documentation have meaningful default values that follow a specific story.
+## Code examples
 
-**Code examples and schemas**  
-Every endpoint documentation comes with examples and also provides schemas listing the available attributes with explanatory text.
+- Every endpoint documentation comes with examples and also provides schemas listing the available attributes with explanatory text.
+- Request and response examples that belong to every endpoint documentation have meaningful default values that follow a specific story.
+- Code examples and explanatory text are visually separated.
```
Comment (cgebken) [inline api-guidelines/global/core-principles/documentation.md:25]: Not sure if I get your point here. Sometimes, having an additional example right below the description in the spec helps to understand the property... So I would not force them to only have them in the examples section...

File: api-guidelines/global/core-principles/documentation.md [hunk 5]
```diff
@@ -1,57 +1,39 @@
...
+- API endpoints and examples are shown in context.
```
Comment (cgebken) [inline api-guidelines/global/core-principles/documentation.md:26]: I do not really get what this is about :( :D

File: api-guidelines/global/core-principles/documentation.md [hunk 6]
```diff
@@ -1,57 +1,39 @@
...
-
-**Familiarize users with the API**  
-We'll cater for experienced users that already have an idea of the endpoints to be used, and at the same time provide the respective structure and information for inexperienced users who need a thorough introduction and/or example use cases.
+## Accessibility and usability
 
-**Discoverable and easy to digest**
-
-- All information is easily discoverable, easy to digest, and prepared in a way that the user can effectively work with it.
+- We provide access to all API endpoints in a central location, some sort of documentation portal at the best, to ensure that users can easily use the API.
+- The documentation is browsable.
```
Comment (cgebken) [inline api-guidelines/global/core-principles/documentation.md:14]: My first suggestion was this:

```suggestion
- The documentation is browsable across all available endpoints.
```

But then I moved on to the next bullet point and figured out that I duplicated content with my suggestion. Actually I would remove this bullet point completely as I do not see the added value of it.

File: api-guidelines/global/core-principles/documentation.md [hunk 7]
```diff
@@ -1,57 +1,39 @@
...
+- A well-structured sidebar ensures that the documentation can be navigated so that all the endpoints can be accessed.
+- We give users the chance to get in touch with us.
```
Comment (cgebken) [inline api-guidelines/global/core-principles/documentation.md:16]: Isn't that covert (even better) in "Quality standards"?

File: api-guidelines/global/core-principles/documentation.md [hunk 8]
```diff
@@ -1,57 +1,39 @@
...
-**Familiarize users with the API**  
-We'll cater for experienced users that already have an idea of the endpoints to be used, and at the same time provide the respective structure and information for inexperienced users who need a thorough introduction and/or example use cases.
+## Accessibility and usability
 
-**Discoverable and easy to digest**
-
-- All information is easily discoverable, easy to digest, and prepared in a way that the user can effectively work with it.
+- We provide access to all API endpoints in a central location, some sort of documentation portal at the best, to ensure that users can easily use the API.
+- The documentation is browsable.
+- A well-structured sidebar ensures that the documentation can be navigated so that all the endpoints can be accessed.
```
Comment (cgebken) [inline api-guidelines/global/core-principles/documentation.md:15]: Somewhat duplicated by "All information is easily discoverable, easy to digest, and prepared in a way that users can effectively work with it." Maybe remove it or at least shorten it to focus on the sidebar?

```suggestion
- A well-structured sidebar ensures that the documentation can be easily navigated.
```

File: api-guidelines/global/core-principles/documentation.md [hunk 9]
```diff
@@ -1,57 +1,39 @@
...
 
-**Automated API reference documentation**  
-API reference documentation is created and published automatically, see [MUST provide API specification using OpenAPI](../../rest/contract/openapi/rules/must-provide-api-specification-using-openapi-for-rest-apis.md).
-:::
+## Context and walkthroughs
 
-::: accordion Consistency and accessibility
+- We give context for API usage such as information about HTTP methods, API operations, request and response headers, versioning, pagination, or errors.
+- With step-by-step walkthroughs we'll cover specific functionality that can be implemented with the API.
+- We cater for experienced users that already have an idea of the endpoints to be used, and at the same time provide the respective structure and information for inexperienced users who need a thorough introduction and/or example use cases.
```
Comment (cgebken) [inline api-guidelines/global/core-principles/documentation.md:33]: Sidenote: We definitely need to update https://github.com/otto-ec/ottoapi_portal/wiki/Language-&-Grammar#andor in order to allow this edge case :D "and or or" would be pretty weird in this case :D

File: api-guidelines/global/core-principles/documentation.md [hunk 10]
```diff
@@ -1,57 +1,39 @@
...
+- API reference documentation is created and published automatically, see [MUST provide API specification using OpenAPI](../../rest/contract/openapi/rules/must-provide-api-specification-using-openapi-for-rest-apis.md).
 
-**Automated API reference documentation**  
-API reference documentation is created and published automatically, see [MUST provide API specification using OpenAPI](../../rest/contract/openapi/rules/must-provide-api-specification-using-openapi-for-rest-apis.md).
-:::
+## Context and walkthroughs
 
-::: accordion Consistency and accessibility
+- We give context for API usage such as information about HTTP methods, API operations, request and response headers, versioning, pagination, or errors.
+- With step-by-step walkthroughs we'll cover specific functionality that can be implemented with the API.
```
Comment (cgebken) [inline api-guidelines/global/core-principles/documentation.md:32]: I would remove that as I don't see us doing that soonish :D

**Review by cgebken — CHANGES_REQUESTED** [review]

File: api-guidelines/global/core-principles/documentation.md (hunk 7 above)
Comment (thake) [reply api-guidelines/global/core-principles/documentation.md:16]: I don't see why. IMHO quality standards is more about the standards we set for our APIs. For me, quality standards has nothing to do with "get into contact".

File: api-guidelines/global/core-principles/documentation.md (hunk 4 above)
Comment (thake) [reply api-guidelines/global/core-principles/documentation.md:25]: I'm with @cgebken on this. The closer the example is to the definition that the example tries to demonstrate, the better.

File: api-guidelines/global/core-principles/documentation.md (hunk 5 above)
Comment (thake) [inline api-guidelines/global/core-principles/documentation.md:26]:
```suggestion
- API endpoints and examples are shown in business context.
```

File: api-guidelines/global/core-principles/documentation.md [hunk 11]
```diff
@@ -1,57 +1,39 @@
...
+- We cater for experienced users that already have an idea of the endpoints to be used, and at the same time provide the respective structure and information for inexperienced users who need a thorough introduction and/or example use cases.
+- We provide an overview of the most likely tasks to perform with the API.
```
Comment (thake) [inline api-guidelines/global/core-principles/documentation.md:34]: Do you have an example of when we are actually doing this? Seems like a legit goal but our API reference tells another story.

File: api-guidelines/global/core-principles/documentation.md [hunk 12]
```diff
@@ -1,57 +1,39 @@
...
-  :::
+## Up-to-dateness
 
-::: accordion Efficient and refined
-The API is well-documented and up-to-date.  
-Users can flawlessly integrate and are informed about recent changes via a revision history.
-:::
-::::
+- The API is well-documented and up-to-date.  
+- Users can flawlessly integrate and are informed about recent changes.
```
Comment (thake) [inline api-guidelines/global/core-principles/documentation.md:39]: At the moment, users can only inform themselves about recent changes. There is no automation to subscribe to changes of one specific endpoint.

File: api-guidelines/global/core-principles/quality-standards.md (hunk 3 above)
Comment (thake) [reply api-guidelines/global/core-principles/quality-standards.md:80]: I also did not know what it meant :sweat_smile:.

**Review by thake — CHANGES_REQUESTED** [review]
Thanks for the effort! Just some minor things. @cgebken made a very thorough review!

File: api-guidelines/global/core-principles/documentation.md (hunk 7 above)
Comment (BirgitBader) [reply api-guidelines/global/core-principles/documentation.md:16]: This one is about giving API consumers the change to get in touch with the providers. Which is what we implemented with "Support" on the API Portal. I would leave it here.

File: api-guidelines/global/core-principles/documentation.md (hunk 4 above)
Comment (BirgitBader) [reply api-guidelines/global/core-principles/documentation.md:25]: Good point. This was actually not related to the spec but to the visualization in the portal. I'll change that.

File: api-guidelines/global/core-principles/documentation.md (hunk 9 above)
Comment (BirgitBader) [reply api-guidelines/global/core-principles/documentation.md:33]: Navigated around it ;-)

File: api-guidelines/global/core-principles/documentation.md (hunk 11 above)
Comment (BirgitBader) [reply api-guidelines/global/core-principles/documentation.md:34]: It's a same as with the walkthroughs... When we wrote this in the past, we've been a bit overambitious ;-) We're not there yet. So I'll remove it.

File: api-guidelines/global/core-principles/documentation.md (hunk 12 above)
Comment (BirgitBader) [reply api-guidelines/global/core-principles/documentation.md:39]: This relates to the changelog. I'll rewrite this.

**Review by cgebken — APPROVED** [review]

File: api-guidelines/global/core-principles/documentation.md (hunk 5 above)
Comment (BirgitBader) [reply api-guidelines/global/core-principles/documentation.md]: I removed it, as discussed in person.

**Pull request merged**